src/core/scene/a-scene.js

```diff
@@ -507,6 +510,80 @@ module.exports.AScene = registerElement('a-scene', {
       writable: true
     },
 
+    setupLoader: {
+      value: function () {
+        var self = this;
```

Comment:
a little daunting to read a web of instantiations that aren't used until later in various places. need to jump up/down the function to follow.